src/kompos/cli/helmfile.py

```diff
@@ -116,15 +117,33 @@ def run(self, args, extra_args):
         return return_code
 
     def setup_kube_config(self, data):
-        if data['helm']['global']['cluster']['type'] == 'eks':
+        if data['helm']['global']['cluster']['type'] == 'k8s':
+            if all(k in data['helm']['global']['cluster']['kubeconfig'] for k in ("path", "context")):
+                if os.path.isfile(data['helm']['global']['cluster']['kubeconfig']['path']):
+                    logger.info('Using kubecofig file: %s', data['helm']['global']['cluster']['kubeconfig']['path'])
+                else:
+                    logger.warning('kubeconfig file not found: %s', data['helm']['global']['cluster']['kubeconfig']['path'])
+                    sys.exit(1)
+
+                kubeconfig_abs_path = os.path.abspath(data['helm']['global']['cluster']['kubeconfig']['path'])
+                conf = KubeConfig(kubeconfig_abs_path)
+                conf.use_context(data['helm']['global']['cluster']['kubeconfig']['context'])
+                os.environ['KUBECONFIG'] = kubeconfig_abs_path
+                logger.info('Current context: %s', conf.current_context())
+            else:
+                logger.warning('path or context keys not found in helm.global.cluster.kubeconfig')
+                sys.exit(1)
```

Comment:
would easier to read if this would be extracted in a separate function like `get k8s_kube_config` and the params would be extracted to separate vars like ```kubeconfig_path = data['helm']['global']['cluster']['kubeconfig']['path']```

Reply:
just a nice to have: a validation of the context inside the kubeconfig file to avoid running against the wrong context